libs/dh/admin/data-access-graphql/project.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "dh-admin-data-access-graphql",
+  "projectType": "library",
+  "sourceRoot": "libs/dh/admin/data-access-graphql/src",
+  "prefix": "dh",
+  "tags": ["product:dh", "domain:admin", "type:data-access"]
+}
```

Comment:
suggestion: since a lib is auto-generated, it's better keep it as is, mainly for consistency with the Nx tooling. The current lib is missing quite a few files. Let's add them and have a discussion at the next guild meeting about which road to go with GraphQL data-access libs.

Reply:
I just copied the way @ManBearTM did it :-)

Reply:
I opted to skip the usual boilerplate since the library contains no TS and the `.graphql` files are never imported, only used by the yarn script. But sure, let’s have a talk about it 🙂

Reply:
I see. Well, the `assets` libs use a similar approach as well. We can use it for GraphQL `data-access` libs too. If we decide to change to something else again in the future, we'll probably generate new libs anyways, so let's use the above convention for GraphQL `data-access` libs.